docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1191,6 +1193,11 @@ async function loadCamera(constraints) {
 
     meetingStreamsLocalVideo.srcObject = localMedia.cameraStream.outputStream;
 
+    localMedia.cameraStream.on('system-mute-state-change', (muted) => {
+      console.log('MeetingControls#loadCamera() :: local camera stream system mute state changed to', muted);
+      handleMuteVideoMessage();
+    });
+
```

Comment:
I guess I can add it for logging purposes. The only reason why we needed it for audio is that the button label changes (we don't change the button label for video... it will always say "Toggle Video").